### PR TITLE
Added more domains

### PR DESCRIPTION
**Pull request by smariconde**

### CODE REVIEW

**Comment by mesuutt** [comment]
Hi @smariconde, thanks for this PR 🚀 


Support for multiple domains is good for us, but this will require permissions to multiple domains and I think someone don't want to allow. Also if user using adblocker, extension not working because when user clicked to button, ajax redirecting to medium.com, so adblocker blocks this request.

How we can handle this stuation? Choices;

- Add these domains to extension.
- Create new extension for multiple domains.
- We can add these domains to readme, if anyone wants to add the domains to extension user would copy urls to manifest and load extension as unpacked.

Do you have any advice?

File: manifest.json [hunk 1]
```diff
@@ -5,7 +5,39 @@
     "version": "2.2",
     "content_scripts": [{
         "js": [ "content.js" ],
-        "matches": [ "https://*.medium.com/*"]
+        "matches": [ 
+        "https://*.medium.com/*",
+        "https://towardsdatascience.com/*",
+        "https://hackernoon.com/*",
```
Comment (volt9801) [inline manifest.json:11]: why do you need to add hackernoon to the list? It doesnt even block you

Comment (smariconde) [reply manifest.json:11]: In reality, I noticed that https://towardsdatascience.com/ was blocking, and I looked in other extensions the domains that medium was using. I just copied that list. I also noticed some weren't blocking, but I decided to add them all. Just a suggestion, I'm new to this world

**Comment by smariconde** [comment]
Hi @mesuutt, Thanks! I just added the list because I noticed there were other domains. Not sure what's the best option. I'm a rookie!

**Pull request closed without merging**